cpg-language-cxx/build.gradle.kts

```diff
@@ -52,4 +52,5 @@ dependencies {
     api(libs.eclipse.cdt.core)
 
     testImplementation(libs.junit.params)
+    testImplementation(project(":cpg-analysis"))
```

Comment:
The checked-in test shouldn't need this dependency

Reply:
They do. we are using `evaluate()`